include/rocksdb/options.h

```diff
@@ -2057,12 +2057,19 @@ struct ReadOptions {
 
   // *** BEGIN options for RocksDB internal use only ***
 
+  // TODO: create a new struct for per-request options, potentially including
+  // timestamps in point lookups/scans
+
+  // request_id a unique id assigned by the application. It is used to allow us
```

Comment:
Mention in the comment that request_id is not necessarily unique to each RocksDB API call. It could (and typically will) be used to identify an application level request that may translate into multiple API calls.